### ISSUE TITLE
Custom Kernel not loading? (Or potential other issue)

### ISSUE THREAD

**Issue by valuta1995** [fabianishere/udm-kernel-tools#33]
Hi,
I'm trying to get KPN IPTV to work through a regular UDM (not pro).

I thought the custom kernel booted successfully, but now I am not so sure. The output for `uname -a` is as follows:
```
Linux Dream-Machine 4.19.152-al-linux-v10.2.0-v1.10.0.3686-a2edd0c #1 SMP Fri Jul 9 03:16:50 UTC 2021 aarch64 GNU/Linux
```

If this is not what I am supposed to see, I think that is the issue to look at; if this is correct however, I have a different issue.

The DHCP debugging seems fine, but the podman logs have a bunch of warnings.
```
# podman logs iptv
udm-iptv: Obtaining IP address for VLAN interface...
udhcpc: started, v1.33.1
udhcpc: sending discover
udhcpc: sending discover
udhcpc: sending select for 10.246.126.144
udhcpc: lease of 10.246.126.144 obtained, lease time 3600
mv: can't rename '/etc/resolv.conf.22': Resource busy
udm-iptv: NATing IPTV network ranges (if necessary)...
iptables: No chain/target/match by that name.
iptables: No chain/target/match by that name.
udm-iptv: Setting up igmpproxy..
udm-iptv: Starting igmpproxy..
MRT_INIT failed; Errno(92): Protocol not available
udm-iptv: Obtaining IP address for VLAN interface...
ifup: interface iptv already configured
udm-iptv: NATing IPTV network ranges (if necessary)...
udm-iptv: Setting up igmpproxy..
udm-iptv: Starting igmpproxy..
MRT_INIT failed; Errno(92): Protocol not available
udm-iptv: Obtaining IP address for VLAN interface...
ifup: interface iptv already configured
udm-iptv: NATing IPTV network ranges (if necessary)...
udm-iptv: Setting up igmpproxy..
udm-iptv: Starting igmpproxy..
MRT_INIT failed; Errno(92): Protocol not available
udm-iptv: Obtaining IP address for VLAN interface...
ifup: interface iptv already configured
udm-iptv: NATing IPTV network ranges (if necessary)...
udm-iptv: Setting up igmpproxy..
udm-iptv: Starting igmpproxy..
MRT_INIT failed; Errno(92): Protocol not available
udm-iptv: Obtaining IP address for VLAN interface...
ifup: interface iptv already configured
udm-iptv: NATing IPTV network ranges (if necessary)...
udm-iptv: Setting up igmpproxy..
udm-iptv: Starting igmpproxy..
MRT_INIT failed; Errno(92): Protocol not available
udm-iptv: Obtaining IP address for VLAN interface...
ifup: interface iptv already configured
udm-iptv: NATing IPTV network ranges (if necessary)...
udm-iptv: Setting up igmpproxy..
udm-iptv: Starting igmpproxy..
MRT_INIT failed; Errno(92): Protocol not available
```

Additionally, the other issue on custom kernel issues asked for the output of `# cat /sys/fs/pstore/*` which is as follows.
```
# cat /sys/fs/pstore/*
[    0.634158] pci-pf-stub 0000:00:04.0: writing to VF config space
[    0.634223] pci-pf-stub 0000:00:05.0: writing to VF config space
[    0.657758] ahci 0001:00:00.0: writing to VF config space
[    0.663257] ahci 0002:00:00.0: writing to VF config space
[    0.668757] ahci 0003:00:00.0: writing to VF config space
[    0.676428] al_eth 0000:00:01.0: writing to VF config space
[    2.634102] al_eth 0000:00:03.0: writing to VF config space
[   14.536743] Unable to handle kernel NULL pointer dereference at virtual address 0000000000000000
[   14.545596] Mem abort info:
[   14.548419]   ESR = 0x96000045
[   14.551508]   Exception class = DABT (current EL), IL = 32 bits
[   14.557485]   SET = 0, FnV = 0
[   14.560564]   EA = 0, S1PTW = 0
[   14.563734] Data abort info:
[   14.566673]   ISV = 0, ISS = 0x00000045
[   14.570537]   CM = 0, WnR = 1
[   14.573535] user pgtable: 4k pages, 39-bit VAs, pgdp = 000000009ac9f6b1
[   14.580203] [0000000000000000] pgd=0000000000000000, pud=0000000000000000
[   14.587022] Internal error: Oops: 96000045 [#1] SMP
[   14.591951] Modules linked in: rlt_wifi(FO+) sch_fq_codel sch_htb lzo lzo_compress zram gpiodev(PFO) ubnthal(PFO) ubnt_common(PFO)
[   14.603740] Process modprobe (pid: 953, stack limit = 0x000000007b4a32a6)
[   14.610551] CPU: 2 PID: 953 Comm: modprobe Tainted: PF          O      4.19.152-edge1 #1
[   14.618690] Hardware name: Annapurna Labs Alpine V2 UBNT (DT)
[   14.624488] pstate: 80000005 (Nzcv daif -PAN -UAO)
[   14.629308] pc : __memcpy+0x94/0x180
[   14.632990] lr : RtmpOSNetDevAttach+0x94/0x140 [rlt_wifi]
[   14.638440] sp : ffffff800c5eb830
[   14.641780] x29: ffffff800c5eb830 x28: 0000000000000100
[   14.647144] x27: ffffff800c61d000 x26: ffffff8008131a98
[   14.652480] x25: ffffff800ca00000 x24: ffffffc075f3e000
[   14.657843] x23: ffffffc07c5ed098 x22: ffffff8000d27000
[   14.663178] x21: ffffff8009945000 x20: ffffffc075f3e000
[   14.668541] x19: ffffff800c5eb8e0 x18: ffffffffffffffff
[   14.673876] x17: 0000000000000001 x16: 0000000000000007
[   14.679211] x15: ffffff8008d59d88 x14: ffffff8008e4bf38
[   14.684574] x13: ffffff8008e4bb85 x12: ffffff8008d73000
[   14.689908] x11: 0000000000000000 x10: 0000000000000000
[   14.695272] x9 : 0000000000000002 x8 : ffffff8008e58567
[   14.700606] x7 : ffffff8008e4b1a8 x6 : 0000000000000000
[   14.705970] x5 : 0000000000000000 x4 : ffffff800c5eb8c8
[   14.711305] x3 : 0000000000000000 x2 : 0000000000000006
[   14.716668] x1 : ffffff800c5eb930 x0 : 0000000000000000
[   14.722003] Call trace:
[   14.724478]  __memcpy+0x94/0x180
[   14.727824]  rt_pci_probe+0x444/0x560 [rlt_wifi]
[   14.732468]  pci_device_probe+0xe4/0x1e0
[   14.736416]  really_probe+0x1e8/0x288
[   14.740133]  driver_probe_device+0x5c/0x100
[   14.744340]  __driver_attach+0xf0/0xf8
[   14.748113]  bus_for_each_dev+0x84/0xd8
[   14.752001]  driver_attach+0x30/0x40
[   14.755601]  bus_add_driver+0x1c4/0x230
[   14.759462]  driver_register+0x64/0x110
[   14.763349]  __pci_register_driver+0x58/0x68
[   14.767690]  rt_pci_init_module+0x164/0x180 [rlt_wifi]
[   14.772927]  wifi_drv_init_module+0x18/0xe80 [rlt_wifi]
[   14.778177]  do_one_initcall+0x68/0x1f8
[   14.782040]  do_init_module+0x64/0x1d8
[   14.785842]  load_module+0x20c8/0x22c0
[   14.789615]  __se_sys_finit_module+0xc4/0xd8
[   14.793908]  __arm64_sys_finit_module+0x24/0x30
[   14.798493]  el0_svc_handler+0xc8/0x1a8
[   14.802354]  el0_svc+0x8/0xc4
[   14.805348] Code: f8408423 f80084c3 36100062 b8404423 (b80044c3)
[   14.811494] ---[ end trace b513d756044bc509 ]---
[   14.819925] Kernel panic - not syncing: Fatal exception
[   14.825202] SMP: stopping secondary CPUs
[   14.829171] Kernel Offset: disabled
[   14.832685] CPU features: 0x0,20006008
[   14.836486] Memory Limit: none
[   14.843368] Rebooting in 3 seconds..

No errors detected
  4.311813] sd 4:0:0:0: [sda] Attached SCSI removable disk
Sep 15 12:57:34 Dream-Machine user.notice kernel: [    4.886396] random: fast init done
Sep 15 12:57:34 Dream-Machine user.notice kernel: [    4.990713] random: crng init done
Sep 15 12:57:34 Dream-Machine user.info kernel: [    5.116596] EXT4-fs (sda6): recovery complete
Sep 15 12:57:34 Dream-Machine user.info kernel: [    5.117183] EXT4-fs (sda6): mounted filesystem with ordered data mode. Opts: errors=panic
Sep 15 12:57:34 Dream-Machine user.warn kernel: [    6.642755] ubnt_common: bad vermagic: kernel tainted.
Sep 15 12:57:34 Dream-Machine user.warn kernel: [    6.642758] Disabling lock debugging due to kernel taint
Sep 15 12:57:34 Dream-Machine user.warn kernel: [    6.642761] ubnt_common: loading out-of-tree module taints kernel.
Sep 15 12:57:34 Dream-Machine user.warn kernel: [    6.642765] ubnt_common: module license 'Proprietary' taints kernel.
Sep 15 12:57:34 Dream-Machine user.info kernel: [    6.646067] [UBNT_STA_HT] Initialize hash table
Sep 15 12:57:34 Dream-Machine user.info kernel: [    6.646091] [UBNT_STA_HT] done
Sep 15 12:57:34 Dream-Machine user.warn kernel: [    6.646093] creating /proc/ubnt_peek/
Sep 15 12:57:34 Dream-Machine user.warn kernel: [    6.646100] creating /proc/ubnt_peek/peek_disable
Sep 15 12:57:34 Dream-Machine user.warn kernel: [    6.646102] creating /proc/ubnt_peek/loop_check
Sep 15 12:57:34 Dream-Machine user.warn kernel: [    6.646105] creating /proc/ubnt_peek/loop_debug
Sep 15 12:57:34 Dream-Machine user.warn kernel: [    6.646107] creating /proc/ubnt_peek/loop_uif
Sep 15 12:57:34 Dream-Machine user.warn kernel: [    6.646108] creating /proc/ubnt_peek/tcp_debug_mac
Sep 15 12:57:34 Dream-Machine user.warn kernel: [    6.646111] creating /proc/ubnt_peek/tcp_debug_connections
Sep 15 12:57:34 Dream-Machine user.warn kernel: [    6.646123] qos: registered
Sep 15 12:57:34 Dream-Machine user.warn kernel: [    6.646125] creating /proc/ubnt_frame_id/
Sep 15 12:57:34 Dream-Machine user.warn kernel: [    6.646128] creating /proc/ubnt_frame_id/netdevs
Sep 15 12:57:34 Dream-Machine user.info kernel: [    6.699131] ubnthal: Ubiquiti UniFi Dream Machine
Sep 15 12:57:34 Dream-Machine user.debug kernel: [    6.700095] creating procfs for ubnthal
Sep 15 12:57:34 Dream-Machine user.debug kernel: [    6.700103] creating proc entry for system.info
Sep 15 12:57:34 Dream-Machine user.debug kernel: [    6.700105] creating proc entry for board
Sep 15 12:57:34 Dream-Machine user.info kernel: [    6.709229] Unable to find matching dts node.
Sep 15 12:57:34 Dream-Machine user.debug kernel: [    6.709236] creating procfs for status
Sep 15 12:57:34 Dream-Machine user.debug kernel: [    6.709241] creating proc entry for IsDefault
Sep 15 12:57:34 Dream-Machine user.debug kernel: [    6.709243] creating proc entry for IsLocated
Sep 15 12:57:34 Dream-Machine user.debug kernel: [    6.709245] creating proc entry for IsIsolated
Sep 15 12:57:34 Dream-Machine user.debug kernel: [    6.709247] creating proc entry for IsLte
Sep 15 12:57:34 Dream-Machine user.debug kernel: [    6.709249] creating proc entry for ControllerPort
Sep 15 12:57:34 Dream-Machine user.debug kernel: [    6.709253] creating proc entry for ControllerHost
Sep 15 12:57:34 Dream-Machine user.info kernel: [    6.842441] EXT4-fs (sda4): mounted filesystem with ordered data mode. Opts: errors=panic
Sep 15 12:57:34 Dream-Machine user.debug kernel: [    6.857576] creating /proc/gpio/
Sep 15 12:57:34 Dream-Machine user.debug kernel: [    6.857584]          /proc/gpio/ledbar_control
Sep 15 12:57:34 Dream-Machine user.debug kernel: [    6.857586]          /proc/gpio/led_pattern
Sep 15 12:57:34 Dream-Machine user.debug kernel: [    6.857588]          /proc/gpio/led_tempo
Sep 15 12:57:34 Dream-Machine user.debug kernel: [    6.857590]          /proc/gpio/poe_passthrough
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.153116] zram: Added device: zram0
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.164815] zram0: detected capacity change from 0 to 3666362368
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.178825] Adding 3580428k swap on /dev/zram0.  Priority:100 extents:1 across:3580428k SS
Sep 15 12:57:34 Dream-Machine user.warn kernel: [    7.346539] ICMPv6: process `sysctl' is using deprecated sysctl (syscall) net.ipv6.neigh.default.base_reachable_time - use net.ipv6.neigh.default.base_reachable_time_ms instead
Sep 15 12:57:34 Dream-Machine user.notice ubios-udm-init: Renaming interfaces ...
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.396005] al_eth 0000:00:01.0 switch0.tmp: renamed from eth0
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.412153] al_eth 0000:00:03.0 eth4.tmp: renamed from eth1
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.434953] al_eth 0000:00:03.0 eth4: renamed from eth4.tmp
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.462515] al_eth 0000:00:03.0: al_mod_eth_function_reset: performing FLR
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.467944] Configured MAC to RGMII mode
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.469071] al_eth 0000:00:03.0 eth4: using MSI-X per Queue interrupt mode
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.472754] IPv6: ADDRCONF(NETDEV_UP): eth4: link is not ready
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.472793] IPv6: ADDRCONF(NETDEV_CHANGE): eth4: link becomes ready
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.478561] al_eth 0000:00:01.0 switch0: renamed from switch0.tmp
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.498521] al_eth 0000:00:01.0: al_mod_eth_function_reset: performing FLR
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.503981] Configured MAC to RGMII mode
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.505098] al_eth 0000:00:01.0 switch0: using MSI-X per Queue interrupt mode
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.508786] IPv6: ADDRCONF(NETDEV_UP): switch0: link is not ready
Sep 15 12:57:34 Dream-Machine user.notice ubios-udm-init: Creating bridge interfaces ...
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.532108] br0: port 1(switch0) entered blocking state
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.532112] br0: port 1(switch0) entered disabled state
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.532216] device switch0 entered promiscuous mode
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.535188] IPv6: ADDRCONF(NETDEV_UP): br0: link is not ready
Sep 15 12:57:34 Dream-Machine user.notice ubios-udm-init: Creating macvlan interfaces ...
Sep 15 12:57:34 Dream-Machine user.notice ubios-udm-init: Creating svlan interfaces ...
Sep 15 12:57:34 Dream-Machine user.notice ubios-udm-init: Creating ethvlan interfaces ...
Sep 15 12:57:34 Dream-Machine user.notice ubios-udm-init: Setting MAC addresses for interfaces (excluding bridges) ..
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.657534] br0: port 1(switch0) entered disabled state
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.660573] al_eth 0000:00:01.0: al_mod_eth_function_reset: performing FLR
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.666024] Configured MAC to RGMII mode
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.667084] al_eth 0000:00:01.0 switch0: using MSI-X per Queue interrupt mode
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.670748] IPv6: ADDRCONF(NETDEV_UP): switch0: link is not ready
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.671399] eth [al_mod_eth_1]: set link speed to 1000Mbps. full duplex.
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.671408] al_eth 0000:00:01.0 switch0: Link is Up - 1Gbps/Full - flow control off
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.694114] al_eth 0000:00:03.0: al_mod_eth_function_reset: performing FLR
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.699576] Configured MAC to RGMII mode
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.700587] al_eth 0000:00:03.0 eth4: using MSI-X per Queue interrupt mode
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.704816] eth [al_mod_eth_3]: set link speed to 1000Mbps. full duplex.
Sep 15 12:57:34 Dream-Machine user.info kernel: [    7.704828] al_eth 0000:00:03.0 eth4: Link is Up - 1Gbps/Full - flow control off
Sep 15 12:57:34 Dream-Machine user.notice ubios-udm-init: Setting mac for bridges ...
Sep 15 12:57:34 Dream-Machine user.notice ubios-udm-init: Apply interface's settings ..
Sep 15 12:57:35 Dream-Machine user.info kernel: [    8.476991] IPv6: ADDRCONF(NETDEV_CHANGE): switch0: link becomes ready
Sep 15 12:57:35 Dream-Machine user.info kernel: [    8.477044] br0: port 1(switch0) entered blocking state
Sep 15 12:57:35 Dream-Machine user.info kernel: [    8.477047] br0: port 1(switch0) entered forwarding state
Sep 15 12:57:35 Dream-Machine user.info kernel: [    8.477106] IPv6: ADDRCONF(NETDEV_CHANGE): br0: link becomes ready
Sep 15 12:57:39 Dream-Machine user.err kernel: [   12.952878] Bluetooth: hci0: unexpected event for opcode 0x0000
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  system: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  resolv: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  syslog: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  netconsole: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  display: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  ledbar: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  rps: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  radio: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  wireless: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  mesh: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  wevent: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  ebtables: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  vlan: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  unifi_block: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  netconf: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  connectivity: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  aaa: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  sshd: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  redirector: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  poe_passthrough: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  lldpd: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  watchdog: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.err ubntconf: ERROR: watchdog: Plugin failed: -2
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  unifi: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  player: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  ble: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  element_adopt: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  mark: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  mtk_ble: Running plugin
Sep 15 12:57:41 Dream-Machine daemon.info ubntconf: INFO:  infrared: Running plugin
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.450502] register rlt_drv
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.535541] rt_pci_probe: pci_set_dma_mask okay!
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.536139]
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.536139]
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.536139] === pAd = 0000000012f3e9e4, size = 2297168 ===
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.536139]
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.536163] <-- RTMPAllocTxRxRingMemory, Status=0, ErrorValue=0x
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.536500] <-- RTMPAllocAdapterBlock, Status=0
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.536503] pAd->CSRBaseAddress =0xffffff800ca00000, csr_addr=0xffffff800ca00000!
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.536507] device_id =0x7603
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.536515] RtmpChipOpsHook(761): Not support for HIF_MT yet!
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.536517] mt7603_init()-->
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.536520] mt_bcn_buf_init(296): Not support for HIF_MT yet!
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.536522] <--mt7603_init()
Sep 15 12:57:41 Dream-Machine user.info kernel: [   14.536541] wevent: registered
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.536543] ra0: NO2G init ACL table
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.536634] 80211> RFICType= 1, NumOfChan= 14
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.536635] 80211> Number of rate = 12
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.536637] 80211> CurTxPower = 0 dBm
Sep 15 12:57:41 Dream-Machine user.warn kernel: [   14.536734] 80211> CFG80211_Register with max interface 8
Sep 15 12:57:41 Dream-Machine user.alert kernel: [   14.536743] Unable to handle kernel NULL pointer dereference at virtual address 0000000000000000
Sep 15 12:57:41 Dream-Machine user.alert kernel: [   14.545596] Mem abort info:
Sep 15 12:57:41 Dream-Machine user.alert kernel: [   14.548419]   ESR = 0x96000045
Sep 15 12:57:41 Dream-Machine user.alert kernel: [   14.551508]   Exception class = DABT (current EL), IL = 32 bits
Sep 15 12:57:41 Dream-Machine user.alert kernel: [   14.557485]   SET = 0, FnV = 0
Sep 15 12:57:41 Dream-Machine user.alert kernel: [   14.560564]   EA = 0, S1PTW = 0
Sep 15 12:57:41 Dream-Machine user.alert kernel: [   14.563734] Data abort info:
Sep 15 12:57:41 Dream-Machine user.alert kernel: [   14.566673]   ISV = 0, ISS = 0x00000045
Sep 15 12:57:41 Dream-Machine user.alert kernel: [   14.570537]   CM = 0, WnR = 1
Sep 15 12:57:41 Dream-Machine user.alert kernel: [   14.573535] user pgtable: 4k pages, 39-bit VAs, pgdp = 000000009ac9f6b1
Sep 15 12:57:41 Dream-Machine user.alert kernel: [   14.580203] [0000000000000000] pgd=0000000000000000, pud=0000000000000000

No errors detected
```

**Comment by fabianishere** [comment]
Hi @valuta1995,

This is a known issue. See https://github.com/fabianishere/udm-kernel-tools/issues/30#issuecomment-907107115 from #30. As described there, my suggestion is to upgrade to beta firmware [v1.11.0+](https://community.ui.com/releases/UniFi-OS-Dream-Machines-1-11-0-14/71916646-d8f6-41c0-b145-2fbe2db7c278) if you only need the custom kernel for multicast routing (for IPTV). 

The stock kernel of that firmware versions supports multicast routing directly, so it is not necessary to boot a custom kernel.

**Comment by valuta1995** [comment]
Hi @fabianishere,

That is what I tried following the guide, but that link is a 404 for me and the internet archive has no back-up for it. Does the link work for you? If so, do I need to be logged in somewhere?

**Comment by fabianishere** [comment]
You can probably only see it if you are logged in and enrolled for the beta program. Does the following "mirror" work for you: https://unifi-forum.nl/onderwerpen/unifi-os-dream-machines-1-10-0-15.2124/
It also contains instruction on how to install the update.

**Comment by valuta1995** [comment]
The "Mirror" does in fact work (using the -base model at the bottom of the OP) I tried enrolling for the beta program, but I still get the following message: https://i.imgur.com/w7SEcio.png Perhaps that takes some time to get updated.

Anyway, the "Mirror" worked, I will try loading that firmware image. Would you recommend cleaning up the installed programs from the guide (or at least disabling the kernel auto-boot?

**Comment by fabianishere** [comment]
I don't expect the installed programs to cause any issues, but if you are not using them, my suggestion would be to remove the packages as follows:

```bash
apt remove udm-kernel*
```

**Comment by valuta1995** [comment]
By the way, did you mean to link https://unifi-forum.nl/onderwerpen/unifi-os-dream-machines-1-11-0-14.2311/ (1.11.0-14) instead of 1.10.0-15 or should the 1.10.0-15 work just as well?

**Comment by fabianishere** [comment]
Version 1.10.0-14 is the first version that supports multicast routing in the stock kernel, so I expect anything above that to work.

**Comment by TheJulianJES** [comment]
I think 1.11.0-14 is the first (and only) version. 1.10.x didn't support it

**Comment by fabianishere** [comment]
@TheJulianJES Thanks for the correction. I mistakenly assumed that Ubiquiti had recently released v1.11.0-15 🤦

@valuta1995 Sorry for the confusion. The link you posted is indeed the firmware version you need (so version 1.**11**.0-14 or above):
https://unifi-forum.nl/onderwerpen/unifi-os-dream-machines-1-11-0-14.2311

**Comment by valuta1995** [comment]
That indeed resolved the issue. I didn't even need to change any settings (beyond uninstalling the superfluous packages)
